infrastructure/main.tf

```diff
@@ -76,7 +77,20 @@ module "key-vault" {
   product_group_object_id = "38f9dea6-e861-4a50-9e73-21e64f563537"
 }
 
+# used in output
```

Comment:
How about `used for passing the secrets to end-to-end tests and other services`? We will need to store the topic connection string here, so that other apps have somewhere to take it form. And tests, of course.